packages/wonder-stuff-core/src/types.ts

```diff
@@ -37,3 +37,28 @@ export type Secrets = {readonly [key: string]: SecretString};
  * Make a read-only type mutable.
  */
 export type Mutable<T> = {-readonly [P in keyof T]: T[P]};
+
+/**
+ * Remove the keys in `R` from `C`.
+ */
+export type Without<C, R> = Omit<C, keyof R>;
```

Comment:
I'm kind of questioning the usefulness of this type.  It's used primarily in components that use HOCs in webapp.  We only had a helper type before because Flow didn't have `Omit`.  I kind of think we should just replace `Without` with `Omit`.

Reply:
I don't mind either way. It's a nice, clear utility that drops the need for the extra `keyof` and saves 2 keystrokes. However, if we didn't have it, I wouldn't mind either - but then you have to go find replace all the uses. Is that worth it? I don't know.

I don't think it does any harm existing.